interface/super/edit_layout.php

```diff
@@ -2112,5 +2158,36 @@ function IsNumeric(value, min, max) {
 function IsN(num) { return !/\D/.test(num); }
 
 </script>
+<!-- Below dialog for field modifiers -->
+<!-- <div id="modifierModal" class="modal fade" role="dialog">
+    <div class="modal-dialog modal-lg">
+        <div class="modal-content">
+            <div class="modal-header" >
+                <button type="button" class="close" data-dismiss="modal">&times;</button>
+                <h4><span class="glyphicon glyphicon-list"></span> Field Add On Modifiers</h4>
+            </div>
+            <div class="modal-body">
+                <form name="modifierList" id="modForm" class="text-center">
+                    <div class="center-block">
+                    <table id="modifierPop" class="table table-condensed table-striped" width='100%'>
+                        <thead>
+                            <tr>
+                                <th>Modifier</th>
+                                <th>Description</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                        </tbody>
+                    </table>
+                    </div>
+                </form>
+            </div>
+            <div class="modal-footer">
+                <button type="button" class="btn btn-primary pull-right"><?php echo xlt('Done');?></button>
+                <button type="button" class="btn btn-danger btn-default pull-left" data-dismiss="modal"><span class="glyphicon glyphicon-remove"></span> Cancel</button>
+            </div>
+        </div>
+    </div>
+</div> -->
```

Comment:
Is above code being used?

Reply:
Left it there for Rod to look at and it will be useful in future development. Is it a huge problem leaving in?

Reply:
You might instead start an issue or chat topic about modals in general. There's certainly a need to standardize them and settle on the best solution. I would like something that can be moved and resized by the user.

Reply:
Will do and that will be an interesting discussion too be sure:).